doc/cephfs/fs-nfs-exports.rst

```diff
@@ -67,7 +67,7 @@ Delete NFS Ganesha Cluster
 
 .. code:: bash
 
-    $ ceph nfs cluster delete <clusterid>
+    $ ceph nfs cluster rm <clusterid>
```

Comment:
Why is this change required?

Reply:
it just makes the CLI consistent; everywhere else we use ``rm`` instead of ``remove`` or ``delete``.  are there any users of this besides the tests?

Reply:
@liewegas can you please update the commit message with the reason for removal ?